perception/sandbox/polygrasp/src/polygrasp/grasp_rpc.py

```diff
@@ -0,0 +1,162 @@
+import logging
+import numpy as np
+import open3d as o3d
+from matplotlib import pyplot as plt
+
+import a0
+
+import graspnetAPI
+from polygrasp import serdes
+from polygrasp.serdes import polygrasp_msgs
+
+import signal
+
+
+log = logging.getLogger(__name__)
+topic_key = "grasp_server"
+grasp_topic_key = f"{topic_key}/grasp"
+collision_topic_key = f"{topic_key}/collision"
+
+
+def save_img(img, name):
+    f = plt.figure()
+    plt.imshow(img)
+    f.savefig(f"{name}.png")
+    plt.close(f)
+
+
+class GraspServer:
+    def _get_grasps(self, pcd: o3d.geometry.PointCloud) -> np.ndarray:
+        raise NotImplementedError
```

Comment:
document function / class - why is this throwing not implemented error? presuming we should overwrite that

Reply:
see third_party dirs